src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/builder/HaFlowBuilder.groovy

```diff
@@ -96,6 +179,15 @@ class HaFlowBuilder {
         new HaFlowExtended(haFlow, northboundV2, topologyDefinition)
     }
 
+    /**
+     * Adds ha-flow and waits for it to become UP.
```

Comment:
Either add waiter for flow to become up, or remove this part from method description.

Reply:
sorry, copy+paste